main.go

```diff
@@ -37,6 +39,11 @@ func setLogLevel(environment env.Env) {
 func runAction(ctx models.Context) error {
 	action := ctx.GetEnv().GetPipelinesAction()
 	log.Infof("execute an action %s", action)
+
+	err := git.ResetGitHead(operatorGit.Workspace, ctx.GetEnv().GetWebhookCommitId())
```

Comment:
`runAction` maibe not the best place to use this method. `ProcessRadixAppConfig` (maybe not good name already now) works with git repo, but `RunPipelineJob` has no git related logic.
Can be move this method call to the  `ProcessRadixAppConfig` (maybe it also can be renamed to something like "PreparePipelineContext")